### ISSUE TITLE
Exceptions while handling exceptions in commit_check

### ISSUE THREAD

**Issue by bcavns01** [Juniper/py-junos-eznc#849]
https://github.com/Juniper/py-junos-eznc/blob/8f8cbaf3677d7f94fded5863a7cbb0f10c64643c/lib/jnpr/junos/utils/config.py#L197-L200

The code above incorrectly assumes that any error that reaches that point will have an xml property.  I'm currently using pyez from within Ansible, and I get the following error:
```
'ConnectClosedError' object has no attribute 'xml'
```
That exception is masking the true error.

I traced that error back to the section of code listed at the top of this post.  I changed line 200 from a `return` to a bare `raise` and was then able to get the true reason for the exception:
```
fatal: [mpr01.xxx01]: FAILED! => {"changed": false, "msg": "Unable to close the configuration: ConnectClosedError(mpr01.xxx01.example.com)"}
```

I would suggest checking for the presence of the `xml` property and simply using a bare `raise` if none is present.  I can open a PR if desired, but I thought Juniper might have their own preferred solution.

**Comment by bcavns01** [comment]
I see it might be easier to have a PR for reference.

As one option, here's my proposed change:
https://github.com/Juniper/py-junos-eznc/pull/850

**Comment by vnitinv** [comment]
Fixed via #850
